gateway-cli/src/main.rs

```diff
@@ -336,8 +337,25 @@ async fn main() -> Result<()> {
         Command::IgnitionCommand { sp, command } => {
             client.ignition_command(sp.type_, sp.slot, command).await?;
         }
-        Command::ComponentActiveSlot { .. }
-        | Command::StartupOptions { .. } => {
+        Command::ComponentActiveSlot { sp, component, set_slot } => {
+            if let Some(slot) = set_slot {
+                client
+                    .sp_component_active_slot_set(
+                        sp.type_,
```

Comment:
What's the significance of the trailing `_` on type?

Reply:
The OpenAPI spec [says this should be `type`](https://github.com/oxidecomputer/omicron/blob/8dc98ac9e6d6fd6f0b177909af7895060eef7f9b/openapi/gateway.json#L975-L977), but `type` is a reserved word, so progenitor [appends an _](https://github.com/oxidecomputer/progenitor/blob/32f2089f4100613a60dd0bc72674e37f0a72966a/progenitor-impl/src/util.rs#L116-L121) when defining the field name.